test/localized/localized.lean

```diff
@@ -25,11 +25,7 @@ example : 2 ↓ 3 = 8 := rfl
 example : 2 ⊖ 3 = 8 := rfl
 example {n m : ℕ} (h : n < m) : n ≤ m := by { success_if_fail { simp [h] }, exact le_of_lt h }
 
-open_locale int
-example : 2 ⊹ 3 = 6 := rfl
-example : 2 ↓ 3 = 8 := rfl
-example : 2 ⊖ 3 = 8 := rfl
-
+run_cmd success_if_fail $ get_localized [`int]
```

Comment:
Could you add a comment indicating that the purpose of this line is to test what happens if you `open_locale` a nonexistent locale? I can imagine a future maintainer being confused (especially if, for example, an `int` locale does actually exist in the future).

Reply:
So maybe change `int` to `this_locale_should_not_exist`?

Reply:
Both suggestions implemented!